src/commonMain/kotlin/org/hisp/dhis/rules/Environment.kt

```diff
@@ -0,0 +1,3 @@
+package org.hisp.dhis.rules
+
+expect fun getEnvironment(): String
```

Comment:
The return type could directly be TriggerEnvironment to avoid the `valueOf` methods.